python-recipes/RAG/02_langchain.ipynb

```diff
@@ -165,12 +152,9 @@
         "import os\n",
         "\n",
         "# Replace values below with your own if using Redis Cloud instance\n",
-        "REDIS_HOST = os.getenv(\"REDIS_HOST\", \"localhost\")\n",
-        "REDIS_PORT = os.getenv(\"REDIS_PORT\", \"6379\")\n",
-        "REDIS_PASSWORD = os.getenv(\"REDIS_PASSWORD\", \"\")\n",
-        "#REDIS_HOST=\"redis-18374.c253.us-central1-1.gce.cloud.redislabs.com\"\n",
-        "#REDIS_PORT=18374\n",
-        "#REDIS_PASSWORD=\"1TNxTEdYRDgIDKM2gDfasupCADXXXX\"\n",
+        "REDIS_HOST = os.getenv(\"REDIS_HOST\", \"localhost\") # ex: \"redis-18374.c253.us-central1-1.gce.cloud.redislabs.com\"\n",
+        "REDIS_PORT = os.getenv(\"REDIS_PORT\", \"6379\")      # ex: 18374\n",
+        "REDIS_PASSWORD = os.getenv(\"REDIS_PASSWORD\", \"\")  # ex: \"1TNxTEdYRDgIDKM2gDfasupCADXXXX\"\n",
         "\n",
         "# If SSL is enabled on the endpoint, use rediss:// as the URL prefix\n",
```

Comment:
extra 's' in rediss.

Reply:
for `rediss://`? this is the correct usage if SSL/TLS is enabled on the db